apps/Learning-Buddy/screens/NewQuizScreen.jsx

```diff
@@ -0,0 +1,354 @@
+                  borderBottomStartRadius: 15,
+                  borderTopStartRadius: 15,
+                  borderTopEndRadius: 15,
+                  fontFamily: 'Poppins',
+                  fontWeight: '600',
+                  fontSize: 20,
+                  backgroundColor: colors.lightGrey
+                }}
+                value={numQuestions.toString()}
+                onChangeText={(text) => setNumQuestions(parseInt(text))}
+                keyboardType='numeric'
+              />
+              <IconButton
+                icon='chevron-down'
+                size={34}
+                iconColor={colors.white}
+                mode='contained'
+                style={localStyles.iconButton}
+                containerColor={colors.lightBlue}
+                onPress={handleDecrement}
+              />
+            </View>
+          </View>
+
+          <View>
+            <Text style={localStyles.title}>Question Types</Text>
+            <View style={localStyles.container}>
+              <BigButton
+                buttonColor={colors.lightGrey}
+                textColor={
+                  selectedQuestionType === 'Multiple Choice'
+                    ? colors.white
+                    : colors.black
+                }
+                content={'Multiple Choice'}
+                onPress={() => handleQuestionTypePress('Multiple Choice')}
+              />
+              <BigButton
+                buttonColor={colors.lightGrey}
+                textColor={
+                  selectedQuestionType === 'True/False'
+                    ? colors.white
+                    : colors.black
+                }
+                content={'True/False'}
+                onPress={() => handleQuestionTypePress('True/False')}
+              />
+              <BigButton
+                buttonColor={colors.lightGrey}
+                textColor={
+                  selectedQuestionType === 'Completion'
+                    ? colors.white
+                    : colors.black
+                }
+                content={'Completion'}
+                onPress={() => handleQuestionTypePress('Completion')}
+              />
+              <BigButton
+                buttonColor={colors.lightGrey}
+                textColor={
+                  selectedQuestionType === 'Matching'
+                    ? colors.white
+                    : colors.black
+                }
+                content={'Matching'}
+                onPress={() => handleQuestionTypePress('Matching')}
+              />
+              <BigButton
+                buttonColor={colors.lightGrey}
+                textColor={
+                  selectedQuestionType === 'Essays'
+                    ? colors.white
+                    : colors.black
+                }
+                content={'Essays'}
+                onPress={() => handleQuestionTypePress('Essays')}
+              />
+            </View>
+          </View>
+          <View>
```

Comment:
Similarly I think this section should be hidden unless we have functionality for it